Zend/zend_compile.c

```diff
@@ -8120,11 +8120,10 @@ void zend_compile_top_stmt(zend_ast *ast) /* {{{ */
 	}
 
 	if (ast->kind == ZEND_AST_STMT_LIST) {
-		zend_ast_list *list = zend_ast_get_list(ast);
-		zend_ast **begin = list->child;
-		zend_ast **end = begin + list->children;
+		zend_ast_list *const list = zend_ast_get_list(ast);
```

Comment:
nit: We don't use postfix const (at least for locals).

Reply:
This deals with two ranges, and I kept accidentally moving `begin`. I slapped const on a few variables so that the compiler would catch my mistakes here. I can remove it if we really care.